src/App.js

```diff
@@ -4,94 +4,182 @@ import styled from 'styled-components';
-  return `rgb(${random(255)}, ${random(255)}, ${random(255)})`;
-};
+const StyledSubTitle = styled.p`
+  font-size: 12px;
+  color: #5e6c84;
+`;
 
-const changeStyling = () => {
-  console.log('The button has been clicked and the width and font family has been changed');
-  document.body.style.width = '30%';
-  document.body.style.backgroundColor = backgroundChange();
-  document.body.style.fontFamily = 'monospace';
-};
+const StyledGrid = styled.div`
+  max-width: 896px;
+  border: 1px solid #c1c7d0;
+`;
 
-const keyPressFunction = e => {
-  switch (e.key) {
-    case 'Enter':
-      console.log('you hit enter');
-      break;
-    case 'd':
-      console.log('you hit d');
-      break;
-    default:
-      console.log('you hit something else');
-  }
-};
+const StyledTitleRow = styled.div`
+  display: flex;
+  color: rgb(0, 0, 255);
+  background-color: #e6fcff;
+  text-align: center;
+  height: 35px;
+  font-weight: bold;
+`;
 
-const App = () => {
-  const [keyPress, setKeyPress] = useState('');
+const StyledPictureRow = styled.div`
+  display: flex;
+  text-align: center;
+  height: 146px;
+`;
 
-  const outputPrinter = e => {
-    keyPressFunction(e);
-    setKeyPress(`you pressed: ${e.key}`);
-  };
+const StyledCol = styled.div`
+  flex: ${props => props.size};
+  border: 1px solid #c1c7d0;
+  padding: 7px;
+`;
 
-  const handleSubmit = e => {
-    e.preventDefault();
-    console.log(`the event is: ${e.type} and this was triggered on:`, e.target);
-  };
+const StyledImage = styled.img`
+  width: 100px;
+  height: 100px;
+  margin: 23px;
+`;
 
-  const confluenceDocs = () =>
-    confluenceLinks.map((confluenceEl, i) => (
-      <StyledLink key={i} href={confluenceEl.link}>
-        {confluenceEl.name}
-      </StyledLink>
-    ));
+const StyledLink = styled.a``;
 
+const App = () => {
   return (
     <StyledApp data-test-id="body">
       <StyledHeader>
-        <StyledParagraph className="text">Lets start automating!</StyledParagraph>
-        <StyledParagraph className="text">But first, we are learning about the DOM!</StyledParagraph>
-        <StyledParagraph className="text">Practicing CSS</StyledParagraph>
-        <StyledLink href="https://www.sky.com/" target="_blank" rel="noopener noreferrer">
-          Sky website
-        </StyledLink>
-        {confluenceDocs()}
-        <button type="button" onClick={changeStyling} data-test-id="change-colour-button">
-          Change color
-        </button>
-        <form onSubmit={handleSubmit}>
-          <button type="submit">Submit</button>
-        </form>
-        <input id="textbox" type="text" onKeyDown={outputPrinter} data-test-id="key-press-input" />
-        <div id="output">{keyPress}</div>
+        <StyledTitle>Test Chapter</StyledTitle>
+        <StyledSubTitle>
+          Created by Field, James (Agile Test Lead), last modified by Avery, Luke (Scrum Master) on May 19, 2022
+        </StyledSubTitle>
+        <StyledGrid>
+          <StyledTitleRow>
```

Comment:
Everything within the `StyledGrid` tags can be deleted and instead, we can call our `confluenceDocs` function like so:
`<StyledGrid>{confluenceDocs()}</StyledGrid>`